src/_sass/components/_toc.scss

```diff
@@ -4,6 +4,15 @@
   display: none;
   right: 0;
 
+
+  @mixin nav-link-not-active-hover {
```

Comment:
Since this doesn't have a special case for active, perhaps just `nav-link-hover`?

Reply:
This name is more clean.